doc/test-framework/scorecard.md

```diff
@@ -1,80 +1,64 @@
 
-The scorecard is configured by a config file that allows configuring both internal and external plugins as well as a few global configuration options. The configuration
-options are listed below.
+## Requirements 
 
-### Flags
+Following some requirements for the operator project which would be  checked by the scorecard. 
 
-While most configuration is done via a config file, there are a few important flags that can be used:
+- Access to a Kubernetes v1.11.3+ cluster
 
-- `--config` string - path to config file (default `<project_dir>/.osdk-scorecard`; file type and extension can be any of `.yaml`, `.json`, or `.toml`). If a config file is not provided and a config file is not found at the default location, the scorecard will exit with an error.
-- `--output`, `-o` string - output format. Valid options are: `text` and `json`. The default format is `text`, which is designed to be a simpler human readable format. The `json` format uses the JSON schema output format used for plugins defined later in this document.
-- `--kubeconfig` string - path to kubeconfig. It sets the kubeconfig internally for internal plugins and sets the `KUBECONFIG` env var to the provided value for external plugins. If an external plugin specifically sets the `KUBECONFIG` env var, the kubeconfig from the specified env var will be used for that plugin instead.
+**For non-SDK operators:**
 
-### Config File
-
-The config file can be in any of the `json`, `yaml`, or `toml` formats as long as the file has the correct extension. As the config file may be extended to allow configuration
-of all `operator-sdk` subcommands in the future, the scorecard's configuration must be under a `scorecard` subsection. See [Example Config File](#example-config-file) for a YAML formatted example.
-
-### Global options
-
-- `output` string - equivalent of the `--output` flag. If this option is defined by both the config file and the flag, the flag's value takes priority
-- `kubeconfig` string - equivalent of the `--kubeconfig` flag. If this option is defined by both the config file and the flag, the flag's value takes priority
-- `plugins` array - this is an array of `plugin` objects as defined below
-
-### Plugin Struct
+- Resource manifests for installing/configuring the operator and custom resources. (see the [Writing E2E Tests][writing-tests] doc for more information on the global and namespaced manifests)
+- Config getter that supports reading from the `KUBECONFIG` environment variable (such as the `clientcmd` or `controller-runtime` config getters). This is required for the scorecard proxy to work correctly.
 
-A plugin object is used to configure plugins. These are the fields of the plugin object:
+**NOTE:** If you would like to use it to check the integration of your operator project with [OLM][olm] then also the [Cluster Service Version (CSV)][olm-csv] file will be required. It is important to highlight that is a requirement just when the `olm-deployed` option is used. 
```

Comment:
maybe rewording as:  "....This is a requirement when the `olm-deployed` option is used."

Reply:
Done 👍